src/firefly/js/visualize/ui/MouseReadoutOptionPopups.jsx

```diff
@@ -101,10 +185,18 @@ class MouseReadoutOptionDialog extends React.Component {
 		var form;
 
 		if (this.props.groupKey==='PIXEL_OPTION_FORM'){
-			form=  <PixelSizeOptionDialogForm  groupKey={this.props.groupKey} fieldKey={this.props.fieldKey}/>;
+			form=  <PixelSizeOptionDialogForm
+				    groupKey={this.props.groupKey}
+					fieldKey={this.props.fieldKey}
+					radioValue={this.props.radioValue}
+			/>;
 		}
 		else {
-			form= <CoordinateOptionDialogForm  groupKey={this.props.groupKey} fieldKey={this.props.fieldKey}/>;
+			form= <CoordinateOptionDialogForm
+				groupKey={this.props.groupKey}
+				fieldKey={this.props.fieldKey}
+				radioValue={this.props.radioValue}
+			/>;
```

Comment:
Better to start his methods with
`const {groupKey,fieldKey,radioValue}= this.props;`